specs/gloas/validator.md

```diff
@@ -119,6 +120,38 @@ any slot during which `is_proposer(state, validator_index)` returns `True`. The
 mechanism to prepare this beacon block and related sidecars differs from
 previous forks as follows
 
+#### Broadcasting `SignedProposerPreferences`
+
+At the beginning of each epoch, a validator MAY broadcast a
```

Comment:
might be worth to clarify that if a validator does not broadcast a preference it means it does not accept trustless bids via p2p since we will anyways reject all bids on the `execution_payload_bid` topic